VERSION.txt

```diff
@@ -1 +1 @@
-2.1-dev
+2.1.0-dev
```

Comment:
Is this required? Our convention is not to specify the .z value when in dev, just the uneven .y.